README.md

```diff
@@ -508,3 +508,10 @@ Triton Backend API.
 [*repeat*](https://github.com/triton-inference-server/repeat_backend)
 backend shows a more advanced example of how a backend can produce
 multiple responses per request.
+
+* The
+[*stateful*](https://github.com/triton-inference-server/stateful_backend)
+backend manages model state tensors on the server-side for the 
```

Comment:
".. backend shows an example of how a backend can manage model state..."

Reply:
edited the branch with the requested change.